etc/recipes/lsst-libffi/build.sh

```diff
@@ -0,0 +1,15 @@
+./configure --prefix=$PREFIX || exit 1
+make || exit 1
+make install || exit 1
+
+cd $PREFIX
```

Comment:
`glib` and `libffi` are verbatim copies of what was in `conda-recipes`, except for the name change.  I will fix them up and PR the cleanup upstream as well.